packages/web-app-ocm/src/views/IncomingInvitations.vue

```diff
@@ -155,7 +155,32 @@ export default defineComponent({
       }
     }
     const isMyProviderSelectedProvider = (p: ProviderSchema) => {
-      return p.domain === new URL(configStore.serverUrl).hostname
+      const toURL = (purl:string) => {
+        const parts = purl.split('://')
+        switch (true) {
+          case parts.length === 1:
+            // the protocol is missing, assume https to make the URL object happy...
+            return new URL('https://' + purl)
+          default:
+            return new URL(purl)
+        }
+      };
+
+      const {host: configStoreHost, port: configStorePort} = toURL(configStore.serverUrl)
+      const {host: providerSchemaHost, port: providerSchemaPort} = toURL(p.domain)
+      const considerations =[
+        // check if the host is the same
+        configStoreHost === providerSchemaHost,
+        // also check the port, multiple instances can run on the same host but not on the same port...
+        configStorePort === providerSchemaPort,
+      ]
+
+      switch (true) {
+        case !considerations.length:
+          return false
+        default:
+          return considerations.every((c) => c)
+      }
```

Comment:
I think you don't need to switch statement since `considerations.length` is always 2, no?

```suggestion
      return considerations.every((c) => c)
```

Reply:
I thought you omitted the switch ?  @fschade :D